networking_ccloud/tools/netbox_config_gen.py

```diff
@@ -213,6 +216,36 @@ def _ensure_single_item(cls, item_set: Iterable[Any], entity: str, identifier: s
                                         f'Unexpected values for {attribute}, expected 1 but got {len(item_set)}')
         return item_set.pop()
 
+    @classmethod
+    def _handle_interface_speed(cls, iface: NbRecord) -> Optional[str]:
+        force_speed = cls.force_speed_tag in {x.slug for x in iface.tags}
+        if not force_speed:
+            return None
```

Comment:
```python
if not any(tag.slug == cls.force_speed_tag for tag in iface.tags):
    return None
```
This would only iterate over the list until the first matching item is found, though performance impact would be negligible with the short lists that we have.

Reply:
make sense